### DISCUSSION TITLE
FluentSelect should use EditContext

### DISCUSSION THREAD

**Discussion by ianclough** [microsoft/fluentui-blazor#1309] [General]
FluentSelect and other sub-classes of ListComponentBase don't inherit from FluentInputBase and hence don't interact with EditContext. You can use the ValueChanged event to notify updates to EditContext but ideally these input components would behave in a similar way to other inputs.

**Comment by vnbaaij** [comment]
Agreed. We have this on the list to investigate (and hopefully implement) for the next minor version (v4.4)

**Reply by Ducki** [reply]
Does this mean, that `FluentSelect` is practically unusable right now in a Blazor SSR context?

**Reply by vnbaaij** [reply]
Yes. As is the case with a lot of components in the library, having interactivity is quite important.

**Reply by snerte** [reply]
I was wondering if this issue was indeed fixed in 4.4?
